src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StocksController.java

```diff
@@ -60,7 +60,7 @@ public String getStocks(@PathVariable String companyNumber,
     public String postStocks(@PathVariable String companyNumber,
                               @PathVariable String transactionId,
                               @PathVariable String companyAccountsId,
-                              @ModelAttribute("stocks") @Valid StocksNote stocksNote,
+                              @ModelAttribute("stocksNote") @Valid StocksNote stocksNote,
```

Comment:
we dont need to add the suffix `Note. I would be tempted to remove it as the notes are already in their own package.

Reply:
it was named after the web model, which is called stocksNote, because it contains stocks.

Reply:
Ignore me..I've just checked the spec